### DISCUSSION TITLE
cyber_channel fail in ubuntu16.04

### DISCUSSION THREAD

**Discussion by truewq** [minhanghuang/CyberRT#8] [Q&A] [answered]
Hi @minhanghuang 
I get this error after build cyberRT on ubuntu16.04, any suggestion about how to solve this problem.
你好，大佬黄
我下载了你抽取的cyberRT在ubuntu16.04上编译成功后，运行程序正常，但是执行python脚本失败，有什么建议解决思路吗？
![image](https://user-images.githubusercontent.com/7168949/206342496-113de6cc-58a4-4831-844f-afcfe357c284.png)

**Comment by minhanghuang** [comment, marked as answer]
你好@truewq, 这个问题似乎是Python3.5的原因, 你可以升级一下Python的版本`3.6`, https://github.com/protocolbuffers/protobuf/issues/10076#issuecomment-1140753570

**Reply by truewq** [reply]
Hi @minhanghuang 
Thx, I fix this problem by following steps:
1. install python3.6.12 from source code
2. solve compatible problem about python3.6 on ubuntu16.04
refer link:
https://bambooxking.github.io/2020/08/26/%E7%96%91%E9%9A%BE%E6%9D%82%E7%97%87/Ubuntu16-04%E5%AE%89%E8%A3%85Python3-6%E8%B8%A9%E5%9D%91%E8%AE%B0%E5%BD%95/

4. remove and install protobuf 3.14.0 by pip3
`sudo pip3 install protobuf==3.14.0`

![image](https://user-images.githubusercontent.com/7168949/206407116-bcfc001d-4bd5-4add-8b25-b425fa7b4f24.png)
